rustls-test/src/lib.rs

```diff
@@ -1283,8 +1283,8 @@ impl ClientCertVerifier for MockClientVerifier {
         self.mandatory
     }
 
-    fn root_hint_subjects(&self) -> &[DistinguishedName] {
-        &self.subjects
+    fn root_hint_subjects(&self) -> Arc<Vec<DistinguishedName>> {
+        Arc::clone(&self.subjects)
```

Comment:
We prefer to avoid the `Arc::clone()` idiom in favor of just doing `self.subjects.clone()` (also in a few other spots).

Reply:
Done